README.md

```diff
@@ -1,16 +1,18 @@
-![CI](https://github.com/peburrows/goth/workflows/CI/badge.svg)
-
 # Goth
 
-<!-- MDOC !-->
+![CI](https://github.com/peburrows/goth/workflows/CI/badge.svg)
+
+<!-- MDOC -->
```

Comment:
Given we include readme in hexdocs, let's skip it from moduledoc. I think that's what I had in mind but I forgot to remove the MDOCs.

```suggestion
```